src/main/java/ca/spottedleaf/starlight/common/light/StarLightInterface.java

```diff
@@ -269,6 +269,8 @@ public int getRawBrightness(final BlockPos pos, final int ambientDarkness) {
         final ChunkAccess chunk = this.getAnyChunkNow(pos.getX() >> 4, pos.getZ() >> 4);
 
         final int sky = this.getSkyLightValue(pos, chunk) - ambientDarkness;
+        // Don't fetch the block light level if the skylight level is 15, since the value will never be higher.
+        if (sky == 15) return 15;
```

Comment:
The codestyle in the rest of this file uses more lines breaks and always uses braces for if statements like this. I can't speak for leaf but I would prefer if this matched better.